paradise_papers_search/templates/index.html

```diff
@@ -111,42 +109,156 @@ <h1 class="display-3 mb-5">Paradise Papers Search</h1>
-                  <tr data-bind="foreach: $parent._nodePropertyList">
-                    <td data-bind="text: $parent[$data.property_name]"></td>
-                  </tr>
-                </tbody>
-              </table>
-              <div class="row search-home">
-                  <div class="col text-center" data-bind="click: fetch">
-                    <div class="icon-div">
+              <div class="table-responsive">
+                <table class="table table-striped table-hover clickable-rows">
+                  <thead>
+                    <!-- For each node property add table header -->
+                    <tr data-bind="foreach: _nodePropertyList">
+                      <th scope="col"
+                          data-bind="text: $data.property_label,
+                                     visible: $data.property_display_table">
+                      </th>
+                    </tr>
+                  </thead>
+                  <!-- For each row in _nodeSearchData add the node property related data -->
+                  <tbody data-bind="foreach: _nodeSearchData">
+                    <tr data-toggle="modal"
+                        data-target="#nodeDetailsModal"
+                        data-node_id=""
+                        data-bind="foreach: $parent._nodePropertyList,
+                                   attr: {'data-node_id': $data._node_properties.node_id},
+                                   click: (node)=>{$root.displayNode(node)}">
+                      <td data-bind="text: $parent._node_properties[$data.property_name],
+                                     visible: $data.property_display_table">
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+              <div class="row">
+                <div class="col text-center load-more" data-bind="click: fetch">
+                  <div class="icon-div">
                     <i data-bind="visible: !_fetchState()" class="fa fa-chevron-down fa-1x"></i>
                     <i data-bind="visible: _fetchState" class="fa fa-circle-o-notch fa-spin fa-1x"></i>
                   </div>
-                    <div >
+                  <div>
                     <!-- ko ifnot: _fetchState -->
-                      Load More
+                    Load More
                     <!-- /ko -->
                     <!-- ko if: _fetchState -->
-                      Loading Nodes
+                    Loading Nodes
                     <!-- /ko -->
-                    </div>
                   </div>
-            </div>
+                </div>
+              </div>
             </div>
           </div>
+
         </div>
       </div>
-
     </div> <!-- Search Results Container  -->
 
 
+    <!-- Node Details Modal  -->
+    <div id="nodeDetailsModal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="nodeDetailsModal" aria-hidden="true">
```

Comment:
What is the purpose of the `aria-lablelleby` here?
Is using the modal own id 🤔

Reply:
Yeah, maybe I can assign an id to each row representing a node, en then change the aria-labelledby dynamically depending on which node details is being displayed. What do you think?